src/options/arith_options.toml

```diff
@@ -410,6 +410,14 @@ name   = "Arithmetic Theory"
   default    = "false"
   help       = "interleave tangent plane strategy for non-linear incremental linearization solver"
 
+[[option]]
+  name       = "arithExp"
+  category   = "expert"
+  long       = "arith-exp"
+  type       = "bool"
+  default    = "true"
+  help       = "enables non-standard extensions of the arithmetic solver, true by default"
```

Comment:
This is not strictly in scope of this PR, but why do we not give the default value of an option with `-h`? Making this part of the help message statically is not very future proof (in case the default value changes).

Reply:
I can remove these.